managed-databases/redis/how-to/connect-to-redis-database-instance.mdx

```diff
@@ -0,0 +1,56 @@
+---
+meta:
+  title: How to connect to a Redis™ Database Instance
+  description: This page explains how to connect to a Redis™ Database Instance using the Redis™ client
+content:
+  h1: How to connect to a Redis™ Database Instance
+  paragraph: This page explains how to connect to a Redis™ Database Instance using the Redis™ client
+tags: postgresql mysql database-instance managed-database database
+dates:
+  validation: 2023-04-18
+  posted: 2023-04-18
+categories:
+  - managed-databases
+  - redis
+---
+
+You can connect to your Redis™ Database Instance using the [Redis™ CLI](https://redis.io/docs/ui/cli/).
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Redis™ Database Instance](/managed-databases/redis/how-to/create-a-database-for-redis)
+</Message>
+
+1. Click **Redis™** under **Managed Databases** on the side menu. A list of your Redis™ Database Instances displays.
+2. Click the Database Instance name or <Icon name="more" /> > **More info** to access the Database Instance information page.
+3. Retrieve the Database Instance IP and port from the Database Instance information section.
+    <Message type="note">
+    Alternatively, you can click the **copy icon** under **Connection** to retrieve the Redis™ CLI connection command. This already includes an IP and port.
+    </Message>
+
+    <Message type="important">
+    You can use only one of your node IP addresses at a time to connect to your Redis™ Database Instance, as the Redis™ CLI does not differentiate Database Instances in standalone from those in cluster mode.
+    </Message>
+4. Run the following command to connect to your Redis™ Database Instance.
+    <Message type="note">
+    In this tutorial, we will use 192.0.2.1 and 6379 as the example IP and port, respectively.
+    </Message>
+
+    ```
+    redis-cli -h 192.0.2.1 -p 6379 --user <redis-user> --askpass --tls --cacert SSL_<redis-cluster-name>.pem
+    ```
+
+    <Message type="note">
+    The command above uses TLS to add an extra layer of security to your connection. The TLS certificate is generated automatically if you set tls_enabled to true. The certificates take on the following name structure: `SSL_redis-<name-of-your-redis-database-instance>.pem`. When using connectors other than redis-cli, you might need to specify the path to your certificate.
+    </Message>
+5. Enter the database password that you defined upon creation.
+
```

Comment:
We can add something about TLS here. 
On our side we support TLS1.2 as well as TLS1.3. It is possible that some client with old libssl can have problem to connect using redis-cli. In that case, they might want to check the libssl versions installed on their machines

Reply:
j'ai ajouté:
Scaleway supports TLS1.2 and TLS1.3. If you use older versions of [`libssl`](https://cppget.org/libssl), you might encounter connexion issues when using redis-cli. If this is the case, we recommend you check the `libssl` versions installed on your local machine and update if necessary.

ok?

Reply:
perfect thanks